pages/common/csvquote.md

```diff
@@ -0,0 +1,20 @@
+# csvquote
+
+> Enables common utilities such as `cut` and `awk` to work correctly with CSV files containing quoted delimiters and newlines.
+> More information: <https://github.com/dbro/csvquote>.
+
+- Replace quoted special characters in normal csv file:
+
+`csvquote {{path/to/data.csv}}`
+
+- Replace quoted special characters in normal tsv file:
+
+`csvquote -t {{path/to/data.csv}}`
+
+- Restore quoted special characters from stdin:
+
+`csvquote -u`
+
+- Quoting an CSV file for `command` and unquoting after processing (Most Common Use Case):
```

Comment:
```suggestion
- Quote an CSV file for `command` and unquote after processing (Most Common Use Case):
```

Reply:
Quote _a_ CSV file. And please remove "(Most Common Use Case)"